File: src/main/java/com/qsoftware/bubbles/command/HealCommand.java
package com.qsoftware.bubbles.command;

import com.qsoftware.bubbles.common.command.Command;
import com.qsoftware.bubbles.common.command.CommandExecutor;
import com.qsoftware.bubbles.entity.player.PlayerEntity;

public class HealCommand implements CommandExecutor {
    @Override
    public boolean execute(Command command, PlayerEntity player, String[] args) {
        player.setHealth(player.getMaxHealth());
        return true;
    }
}

File: src/main/kotlin/qtech/bubbles/command/HealCommand.kt
package qtech.bubbles.command

import qtech.bubbles.common.command.CommandExecutor
import qtech.bubbles.entity.player.PlayerEntity

open class HealCommand : CommandExecutor {
    override fun execute(player: PlayerEntity, args: Array<String>): Boolean {
        player.damageValue = player.maxDamageValue
        return true
    }
}